EIPS/eip-7718.md

```diff
@@ -48,40 +46,52 @@ Currently defined mainnet protocol identifiers:
-- `0x504E`: Execution Verkle State Network
-- `0x504F`: Execution Transaction Gossip Network
+- `0x504D`: Execution Canonical Transaction Index Network (planned but not implemented)
+- `0x504E`: Execution Verkle State Network (planned but not implemented)
+- `0x504F`: Execution Transaction Gossip Network (planned but not implemented)
+
+### Nodes and Node IDs
+
+Nodes in the portal network are represented by their [EIP-778 Ethereum Node Record (ENR)](./eip-778.md) from the Discovery v5 network. A node's `node-id` is derived according to the node's identity scheme, which is specified in the node's ENR. A node's `node-id` represents its address in the DHT.  Node IDs are interchangeable between 32 byte identifiers and 256 bit integers.
+
 
 ### Content Keys and Content IDs
 
-Content keys are used to request or offer specific content data. As such the content key and content data can be represented as a key:value pair.
+Content keys are used to request or offer specific content data. As such the content key and content data can be thought of as a key-value pair with nodes in the network storing the content data and the content key being the identifier used to request and retrieve the data.
 
-Content keys are passed as byte strings to the messages defined in the Portal wire protocol. How they are encoded is defined per content network specification.
+Each sub-protocol defines the set of supported content keys and the corresponding data payloads for each content key.  The encoding of content keys is defined at the sub-protocol level.
 
-Content IDs are derived from the content keys and are used to identify where the content is located in the network. The derivation is defined per content network specification.
+Content keys are passed in their encoded format as byte strings to the messages defined in the Portal wire protocol.
+
+Content IDs are derived from the content key.  The Content ID can be represented interchangeably as either a 32 byte value or a 256 bit integer.  The Content ID defines the address of the content in the DHT.  The function for deriving the Content ID from a content key is defined at the sub-protocol level.
 
 #### SHA256 Content ID Derivation Function
 
-The SHA256 Content ID derivation function is defined as:
+The SHA256 Content ID derivation function is defined here for convenience as it is the most commonly used:
 
 ```
-content_id = sha256(content_key)
+content_id = sha256(encoded_content_key)
 ```
 
-### Nodes and Node IDs
+### Transmission of data that exceeds the UDP packet limit
+
+The transmission of data that is too large to fit a single packet is done using [uTP](../assets/eip-7718/bep_0029-rst_post.pdf).
+
+> The Portal wire protocol currently implements uTP over the `TALKREQ/TALKRESP` messages.  Future plans are to move to the [sub-protocol data transmission](https://github.com/ethereum/devp2p/issues/229) in order to use a protocol native mechanism for establishing packet streams between clients.
```

Comment:
It has been a while that I looked into that but iirc it is rather about out of band key session setup and not the actual stream/reliable transport setup.